scripts/scr_load/scr_load.gml

```diff
@@ -13,9 +13,29 @@ function scr_load(argument0, argument1) {
 
 
 
-	if (argument0=1) or (argument0=0){debugl("Loading slot "+string(argument1));
-	    file_copy("save"+string(argument1)+".ini","tsave.ini");
-	    file_decrypt("tsave.ini","p");
+	if (argument0=1) or (argument0=0){
```

Comment:
I'll do a refactor of save, load scripts and objects during ironman feature implementation